### PR TITLE
Added mechanism to add simple text command

### LINKED ISSUE #14: Fix warnings in CI build
Check and fix warnings currently in the CI build
https://github.com/discontinuedtv/botbrown/actions/runs/243597141

### PR DESCRIPTION
**Pull request by i-am-steve-oh**

### CODE REVIEW

File: BotBrownCore/Configuration/GreetingConfiguration.cs [hunk 1]
```diff
@@ -3,15 +3,15 @@
     using System.Collections.Generic;
     using System.ComponentModel;
 
-    public class GreetingConfiguration : IConfiguration
+    public class GreetingConfiguration : IChangeableConfiguration
     {
-        public Dictionary<string, string> Greetings { get; set; } = new Dictionary<string, string>();
-
+        public Dictionary<string, string> Greetings { get; set; }
```
Comment (discontinuedtv) [inline BotBrownCore/Configuration/GreetingConfiguration.cs]: Why did you change this from an initialized Dictionary? I don't see a constructor.

File: BotBrownCore/Configuration/UsernameConfiguration.cs [hunk 2]
```diff
@@ -4,15 +4,15 @@
     using System.Collections.Generic;
     using System.ComponentModel;
 
-    public sealed class UsernameConfiguration : IConfiguration
+    public sealed class UsernameConfiguration : IChangeableConfiguration
     {
-        public Dictionary<string, ChannelUser> Users { get; set; } = new Dictionary<string, ChannelUser>();
```
Comment (discontinuedtv) [inline BotBrownCore/Configuration/UsernameConfiguration.cs:9]: Same for this dictionary. In the SimpleTextCommand you added an init method. Should we add this here, too? But isn't the initialization right at here the better option as this property is accessible from outside?

**Review by discontinuedtv — APPROVED** [review]
Looks good!

**Comment by i-am-steve-oh** [comment]
Did you had a chance to look at the commit after your review?
